tests/test_database.py

```diff
@@ -233,3 +235,7 @@ def test_companion_relationships(db):
     assert (
         len(t) == n_companion_relationships
     ), f"Found {len(t)} entries in the Companion Relationships table, expected {n_companion_relationships}"
+
+
+#def test_companion_parameters(db):
```

Comment:
Did you want to write out the test in this PR or just make a ticket for it?

Reply:
done.